packages/react-performance/README.md

```diff
@@ -50,36 +86,131 @@ export function LastNavigationDetails() {
+
+### Parsing report JSON server-side
+
+#### Using Node (`@shopify/koa-performance`)
+
+For Node services based on `Koa`, we provide [@shopify/koa-performance](https://www.npmjs.com/package/@shopify/koa-performance) to parse `PerformanceReport` payloads and send them forward to a [StatsD](https://github.com/statsd/statsd) endpoint as `distribution`s.
+
+#### Using Rails (`quilt_rails`)
+
+For Rails services we provide [quilt_rails](https://www.npmjs.com/package/@shopify/koa-performance)'s `Quilt::Performance::Reportable` mixin for parsing `PerformanceReport` payloads and sending them forward to a [StatsD](https://github.com/statsd/statsd) endpoint as `distribution`s.
+
+### Tracking _usable_ vs _complete_
+
+In a more complex application with multiple pages and calls for remote data, it is helpful to use multiple `<PerformanceMark />` components to track data for when the page is _usable_ vs _complete_. As a rule of thumb, consider a page _usable_ when the top-level React component for it has mounted, and _complete_ when all asynchronous work (such as data fetching) required for rendering the page content has been completed.
+
+For example, we can update our `<Cats />` page component from above and change it to simulate fetching data from a remote endpoint.
+
+```tsx
+// Cats.tsx
+
+import React, {useState, useEffect} from 'react';
+import {PerformanceMark} from '@shopify/react-performance';
+
+/**
+ *  Represents an entire page of our application
+ */
+export function Cats() {
+  // simulate getting data from a remote API
+  const data = useFakeRemoteData();
+
+  return (
+    <section>
+      <h1>Cats</h1>
+      {/* mark the page as usable immediately upon rendering */}
+      <PerformanceMark stage="usable" id="cats" />
+      {data && (
+        <>
+          {/* mark the page as *complete* once we have our data */}
+          <PerformanceMark stage="complete" id="cats" />
+          {data}
+        </>
+      )}
+    </section>
+  );
+}
+
+function useFakeRemoteData() {
+  const [cats, setCats] = useState<string[] | null>(null);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setCats(['Kokusho', 'The Dude', 'Smoky', 'Moe']);
+    }, 1000);
+  });
+
+  return cats;
+}
+```
+
+With these changes we will see separate timing information for `usable` and `complete` in our `usePerformanceReport` output.
 
 ## API
 
 Most APIs provided by this package are available as both hooks (ie. `useNavigationListener`), as well as components (ie. `<NavigationListener />`).
 
-This package also re-exports all of the API of [`@shopify/performance`](https://www.npmjs.com/package/@shopify/performance).
+This package also re-exports all of the API from [`@shopify/performance`](https://www.npmjs.com/package/@shopify/performance).
```

Comment:
aww thats what you are going for before. Both work really, but to keep consistent I will suggest the following.

```suggestion
This package also re-exports all of the API from [`@shopify/performance`](../performance/README.md).
```

Reply:
Don't link directly to the README, but agreed to link to the local package directory